src/Function.FromCallback.tt

```diff
@@ -57,7 +57,12 @@ foreach (var (hasCaller, resultCount, parameterCount, methodGenerics, delegateTy
             var parameterKinds = new List<ValueKind>();
             var resultKinds = new List<ValueKind>();
 
+            <# if (callbackParameterTypeExpressions.Length == 0) { #>
+            var callbackParameterTypes = Array.Empty<Type>();
+            <# } else { #>
             var callbackParameterTypes = new Type[] { <#= callbackParameterTypeExpressions #>};
+            <# } #>
```

Comment:
nit: Note that this syntax causes an additional indentation for the line in the generated `.cs` file:
```diff
-            var callbackParameterTypes = new Type[] { typeof(T), };
+                        var callbackParameterTypes = new Type[] { typeof(T), };
```

Reply:
Thanks, fixed 👍